CHANGELOG.md

```diff
@@ -7,10 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+[Unreleased]: https://github.com/SumoLogic/sumologic-otel-collector/compare/v0.49.0-sumo-0...main
+
+## [v0.49.0-sumo-0]
+
+### Highlights
```

Comment:
Let's skip this header like we did for first release :upside_down_face:

Reply:
Changed it to a `Released at` header for consistency with previous releases.